modules/veteran_verification/spec/requests/metadata_request_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'Metadata Endpoint', type: :request do
+  describe '#get /metadata' do
+    it 'returns metadata JSON for address validation' do
+      get '/services/veteran_verification/address_validation_metadata'
+      expect(response).to have_http_status(:ok)
+      JSON.parse(response.body)
```

Comment:
Personally I think specs are clearer when they have expectations, and if you expect it to not raise an exception there's an expectation to do that!
```
expect { JSON.parse(response.body) }.not_to raise_error
```

Reply:
fixed this